framework/lib/components/date-picker/components/calendar/quick-navigation/get-quick-select-options.ts

```diff
@@ -250,17 +258,18 @@ export const getQuickSelectOptions = (
     fiscalStartMonth === 0 ? [] : [ thisFiscalYear, lastFiscalYear ]
 
   const quickDates = [
-    thisWeek,
     lastWeek,
-    thisMonth,
+    nextWeek,
+    thisWeek,
     lastMonth,
-    thisYear,
-    yearToDate,
-    lastYear,
-    nextMonth,
     nextThreeMonths,
     nextSixMonths,
+    nextMonth,
+    thisMonth,
+    lastYear,
     nextYear,
+    thisYear,
+    yearToDate,
     ...fiscalYears,
   ]
```

Comment:
Has this order been approved by Kate?

Reply:
I talked with Kate this morning and she approved the sorting 🙂